python/mathsys/bin/web/compile.sh

```
@@ -1,55 +1,44 @@
-wasm-ld -flavor wasm -r all.wasm -o all.o
-rm all.wasm
-
-# COMPILE -> RETURN
-cd ..
-cd ..
-cd ..
-cd ..
\ No newline at end of file
+# COMPILE -> COMMAND
+(
+    cd python/mathsys/bin/web
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)